hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/SubProcessHapiTestNode.java

```diff
@@ -35,27 +52,82 @@
  * The {@code stdout} and {@code stderr} files will be written into the working directory.
  */
 final class SubProcessHapiTestNode implements HapiTestNode {
+    private static final Pattern PROM_PLATFORM_STATUS_HELP_PATTERN =
+            Pattern.compile("# HELP platform_PlatformStatus (.*)");
+    private static final Pattern PROM_PLATFORM_STATUS_PATTERN =
+            Pattern.compile("platform_PlatformStatus\\{.*\\} (\\d+)\\.\\d+");
     private static final String[] EMPTY_STRING_ARRAY = new String[0];
     /** The Hedera instance we are testing */
     private ProcessHandle handle;
```

Comment:
Where do we assign this variable? I can only see the assignment in the `start` method(`handle = builder.start().toHandle())` but before that we have the following check `if (handle != null) throw new IllegalStateException("Node is not stopped, cannot start it!");` which means that it will never go there.

Reply:
It is set to null in terminate